core-bundle/src/Resources/contao/drivers/DC_Table.php

```diff
@@ -4512,10 +4520,9 @@ protected function parentView()
 					if ($blnHasSorting)
 					{
 						$security = System::getContainer()->get('security.helper');
-						$subject = new DataContainerSubject($this->strTable);
 
 						// Create new button
-						if (!($GLOBALS['TL_DCA'][$this->strTable]['config']['closed'] ?? null) && !($GLOBALS['TL_DCA'][$this->strTable]['config']['notCreatable'] ?? null) && $security->isGranted(ContaoCorePermissions::DC_ACTION_CREATE, $subject))
+						if (!($GLOBALS['TL_DCA'][$this->strTable]['config']['closed'] ?? null) && !($GLOBALS['TL_DCA'][$this->strTable]['config']['notCreatable'] ?? null) && $security->isGranted(ContaoCorePermissions::DC_PREFIX . $this->strTable, new CreateAction($this->strTable)))
```

Comment:
This `CreateAction` should at least get the PID of the parent record. I guess we don't have a way to also include the position (previous record ID)?

Reply:
Not sure how to achieve this, PR welcome

Reply:
`$row[$i]['id']` would be the new parent ID

Reply:
`$row[$i]['id']` would be the new parent ID